### DISCUSSION TITLE
run on readthedocs

### DISCUSSION THREAD

**Discussion by xmatthias** [LukeCarrier/mkdocs-drawio-exporter#39] [Ideas]
Is it possible to build documentation including this plugin on readthedocs?

i've tried - but (for obvious reasons) it's failing with `Error: drawio-exporter: value "None" for key "drawio_executable" is invalid: Unable to find Draw.io executable; ensure it's on PATH or set drawio_executable option` - which is obvoius if we think about it.

While readthedocs is able to install python dependencies - i'm not aware that this would also allow to install the drawio binaries.

**Comment by LukeCarrier** [comment]
I don’t see an obvious way to ship the needed draw.io binary to Read the Docs either.

In the long run I plan to port this plugin to using [the JavaScript/Puppeteer library](https://github.com/LukeCarrier/drawio-export-puppeteer), but I doubt the RtD environment ships the necessary browsers.

I’ll keep this open for consideration when that work happens, but I can’t provide an ETA I’m afraid.
